config/models.php

```diff
@@ -281,6 +281,16 @@
 
         'except' => [
             'migrations',
+            'failed_jobs',
+            'password_resets',
+            'personal_access_tokens',
```

Comment:
Happy adding the first 3 here, but everything after `personal_access_token` is not included in the base Laravel install. If you need to exclude these tables then they should be added in your published copy of this config.

It would also be worth adding `password_reset_tokens` as the table name was changed in Laravel 10.

Reply:
Done!